src/main/java/org/cactoos/http/HtWire.java

```diff
@@ -102,4 +132,36 @@ public Input send(final Input input) throws IOException {
         }
     }
 
+    /**
+     * Get port number from URI.
+     * @param uri URI to get the port from
+     * @return Port number
+     */
+    private static int prt(final URI uri) {
```

Comment:
Please read [design principles](http://www.elegantobjects.org/#principles) of this project.
One of the rules say "No static methods, not even private ones". So I can't accept PR while there are static methods.
Also, removing of `static` keyword is not an acceptable solution.
For example, this method can be inlined.
`socket()` method also can be inlined because you don't need to wrapping `IOException` into `IllegalStateException` for two reasons:
1. You call to supplier inside of method which already throws `IOException`.
2. You can use `org.cactoos.scalar.CheckedScalar` interface instead of `BiFunction`.